packages/sommelier.rb

```diff
@@ -320,6 +393,8 @@ def self.build
         # stopsommelier
         File.write 'stopsommelier', <<~STOPSOMMELIEREOF
           #!/bin/bash
+          # Quickest way is to use killall on Xwayland
+          pgrep Xwayland &>/dev/null && killall Xwayland
           SOMM="$(pgrep -fc sommelier.elf 2> /dev/null)"
           if [[ "${SOMM}" -gt "0" ]]; then
             pkill -f sommelier.elf &>/dev/null
```

Comment:
Replace `&> /dev/null` with `2>> #{CREW_PREFIX}/var/log/sommelier.log`?

Reply:
That will only put the PID of Xwayland into the log...  is that useful?

Reply:
What happens if there is an actual error?

Reply:
made the change...